docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,23 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned as a row for each table's column, plus a summary row for
+each table. The following table lists the returned columns and what statistics.
```

Comment:
Maybe we can add a sentence about how to identify the summary row (it's unclear currently what is a summary row)?

Reply:
The easiest identifier seems to be `NULL` in `column_name` so I clarified that.